### ISSUE TITLE
rECItoECEF does not work

### ISSUE THREAD

**Issue by thetazero** [sisl/SatelliteDynamics.jl#13]
```julia
using SatelliteDynamics
t=Epoch(2022,7,25,14,40,0)
rECItoECEF(t)
```
Results in:
```julia
ERROR: KeyError: key 59785 not found
Stacktrace:
  [1] getindex
    @ ./dict.jl:482 [inlined]
  [2] UT1_UTC(eop::EarthOrientationData, mjd::Float64; interp::Bool)
    @ SatelliteDynamics ~/.julia/packages/SatelliteDynamics/yvFXe/src/universe.jl:197
  [3] UT1_UTC(mjd::Float64; interp::Bool)
    @ SatelliteDynamics ~/.julia/packages/SatelliteDynamics/yvFXe/src/universe.jl:201
  [4] UT1_UTC
    @ ~/.julia/packages/SatelliteDynamics/yvFXe/src/universe.jl:201 [inlined]
  [5] time_system_offset(jd::Int64, fd::Float64, tsys_src::String, tsys_dest::String)
    @ SatelliteDynamics ~/.julia/packages/SatelliteDynamics/yvFXe/src/time.jl:730
  [6] time_system_offset
    @ ~/.julia/packages/SatelliteDynamics/yvFXe/src/time.jl:741 [inlined]
  [7] mjd(epc::Epoch; tsys::String)
    @ SatelliteDynamics ~/.julia/packages/SatelliteDynamics/yvFXe/src/time.jl:497
  [8] earth_rotation
    @ ~/.julia/packages/SatelliteDynamics/yvFXe/src/reference_systems.jl:250 [inlined]
  [9] rECItoECEF(epc::Epoch)
    @ SatelliteDynamics ~/.julia/packages/SatelliteDynamics/yvFXe/src/reference_systems.jl:299
 [10] top-level scope
    @ REPL[5]:1
```

**Comment by duncaneddy** [comment]
This isn't an issue in the ECI to ECEF transformation, but by the underlying `Epoch` class trying to access Earth Orientation parameter data for a data which isn't present in the packaged EOP data files.

I can update the package data to be more current, but an immediate workaround would be to clone the package locally, then execute the [update_data.jl](https://github.com/sisl/SatelliteDynamics.jl/blob/master/scripts/update_data.jl) script to update the package data. You should find that the built-in data files will be updated. In particular you'd expect to find a line entry with a Modified Julian Date value of `59785` in https://github.com/sisl/SatelliteDynamics.jl/blob/master/data/FINALS.ALL_IAU2000.txt.

**Comment by duncaneddy** [comment]
I've updated the data on the main branch, so if you directly clone the main branch it will work. I'll cut a v0.4.4 once your https://github.com/sisl/SatelliteDynamics.jl/pull/12 is merged.

> ```julia
> using SatelliteDynamics
> t=Epoch(2022,7,25,14,40,0)
Epoch(2022-07-25T14:39:23.000Z)

> rECItoECEF(t)
3×3 Matrix{Float64}:
 -0.955514     0.294937    0.00206812
 -0.294937    -0.955517    0.000659436
  0.00217062   2.01362e-5  0.999998
> ```

**Comment by thetazero** [comment]
Thanks!

Just cloned the main branch and it works for me. Fixed my typo in #12 as well.
